### PR TITLE
1 line fix of rmrf

### PR DESCRIPTION
**Pull request by rob-c**
1 line fix of rmrf

### CODE REVIEW

File: python/Ganga/Core/GangaRepository/GangaRepositoryXML.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def rmrf(name, count=0):
         try:
             remove_name = name
             if not remove_name.endswith('__to_be_deleted'):
-                remove_name = name + "_" + str(time.time()) + '__to_be_deleted_'
+                remove_name += '_%s__to_be_deleted_' % str(time.time())
```
Comment (milliams) [inline python/Ganga/Core/GangaRepository/GangaRepositoryXML.py]: No need for the explicit `str()`. `%s` is performing that action already.

**Comment by rob-c** [comment]
@milliams Are we making use of the git plugins for Jenkins or do we do the git clone in our test scripts? This is failing quite a large amount atm and I'm just wondering if we can wrap it in a try/catch of some sort ?

**Comment by rob-c** [comment]
retest this please

**Comment by milliams** [comment]
We are using the built-in Jenkins system. The fault most likely lies with CERN's networking.

**Comment by rob-c** [comment]
retest this please

**Pull request merged**